src/features/defi/components/Withdraw/Withdraw.tsx

```diff
@@ -95,6 +96,7 @@ export const Withdraw: React.FC<WithdrawProps> = ({
   isLoading,
   percentOptions,
   children,
+  onInputChange = undefined,
```

Comment:
You shouldn't need to default to undefined, it will already be undefined if no `onInputChange` is passed down, could you remove in the follow-up PR?

Reply:
Resolving all comments to be able to merge the PR